client.go

```diff
@@ -400,12 +411,48 @@ func (c *Client) Mail(from string, opts *MailOptions) error {
 // A call to Rcpt must be preceded by a call to Mail and may be followed by
 // a Data call or another Rcpt call.
 //
+// If opts is not nil, RCPT arguments provided in the structure will be added
+// to the command. Handling of unsupported options depends on the extension.
+//
 // If server returns an error, it will be of type *SMTPError.
-func (c *Client) Rcpt(to string) error {
+func (c *Client) Rcpt(to string, opts *RcptOptions) error {
 	if err := validateLine(to); err != nil {
 		return err
 	}
-	if _, _, err := c.cmd(25, "RCPT TO:<%s>", to); err != nil {
+	cmdStr := "RCPT TO:<%s>"
```

Comment:
Is it safe to just append content to `cmdStr`? The new content could have user-supplied `%v` sequences, which `fmt` will choke on.

We should probably set `cmdStr = "RCPT TO:<%s>%s"` and have a separate `extStr` string.

Reply:
It applies to MAIL FROM too which uses similar code.